webapp/src/Controller/API/ContestController.php

```diff
@@ -371,11 +371,11 @@ public function textAction(Request $request, string $cid): Response
             throw new NotFoundHttpException(sprintf('Object with ID \'%s\' not found', $cid));
         }
 
-        if (!$contest->getContestTextType()) {
+        if (!$contest->getContestProblemsetType()) {
             throw new NotFoundHttpException(sprintf('Contest with ID \'%s\' has no text', $cid));
```

Comment:
```suggestion
            throw new NotFoundHttpException(sprintf('Contest with ID \'%s\' has no problemset text', $cid));
```

Not sure if we want this, but just to mark the option.